plugins/modules/azure_rm_roleassignment.py

```diff
@@ -22,16 +23,20 @@
     - Create and delete instance of Azure Role Assignment.
 
 options:
-    name:
-        description:
-            - Unique name of role assignment.
-            - The role assignment name must be a GUID, sample as "3ce0cbb0-58c4-4e6d-a16d-99d86a78b3ca".
-        required: True
     assignee_object_id:
         description:
             - The object id of assignee. This maps to the ID inside the Active Directory.
             - It can point to a user, service principal or security group.
             - Required when creating role assignment.
+    id:
```

Comment:
@paultaiton This ID should be “xxx-assign-guid-xxx",  if it is a complete ID string, then this Value is too long. We can splice it into a complete ID number in the module（Same operation with azure_rm_roleassignment_info module!）. Thank you!

Reply:
The id parameter is the role_assignment_id, which is an full qualified resource id, not a guid. I am not understanding the rest of your suggestion message. What line are you referring to when saying it's too long? How would I go about breaking it up?

Reply:
Yes, You are right! This ID should be an full qualified resource ID! Thank you very much!